chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/DeveloperController.java

```diff
@@ -162,26 +165,46 @@ public ResponseEntity<Developer> update(@PathVariable("developerId") Long develo
+            })
+    @RequestMapping(value = "/{developerId}/join", method = RequestMethod.PUT, consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = "application/json; charset=utf-8")
+    public ChplOneTimeTrigger join(@PathVariable("developerId") Long developerId,
+            @RequestBody(required = true) JoinDevelopersRequest joinRequest)
+            throws InvalidArgumentsException, EntityCreationException, EntityRetrievalException, JsonProcessingException,
+            ValidationException, SchedulerException {
+        if (CollectionUtils.isEmpty(joinRequest.getDeveloperIds())) {
+            throw new InvalidArgumentsException(msgUtil.getMessage("developer.join.missingDeveloperIds"));
+        }
+        return developerManager.join(developerId, joinRequest.getDeveloperIds());
+    }
+
+    @Deprecated
+    @DeprecatedApi(friendlyUrl = "/developers/merge", httpMethod = "POST", removalDate = "2023-10-01",
+            message = "Developer merge is deprecated and will be removed. "
+                    + "Going forward, PUT to /developers/{developerId}/join .")
     @Operation(summary = "Merge developers.",
-            description = "If multiple developer IDs are passed in, the service performs a merge "
-                    + "meaning that a new developer is created with all of the information provided (name, address, "
-                    + "etc.) and all of the products previously assigned to the specified developerId's are "
-                    + "reassigned to the newly created developer. The old developers are then deleted.\n"
-                    + "Security Restrictions: ROLE_ADMIN, ROLE_ONC, or ROLE_ACB if all developers involved are active.",
+            description = "Please use /developers/{developerId}/join .",
             security = {
                     @SecurityRequirement(name = SwaggerSecurityRequirement.API_KEY),
                     @SecurityRequirement(name = SwaggerSecurityRequirement.BEARER)
             })
     @RequestMapping(value = "/merge", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE,
             produces = "application/json; charset=utf-8")
-    public ChplOneTimeTrigger merge(@RequestBody(required = true) MergeDevelopersRequest mergeRequest)
+    public ResponseEntity<Void> merge(@RequestBody(required = true) MergeDevelopersRequest mergeRequest)
             throws InvalidArgumentsException, EntityCreationException, EntityRetrievalException, JsonProcessingException,
             ValidationException, SchedulerException {
-        if (mergeRequest.getDeveloperIds().size() <= 1) {
-            throw new InvalidArgumentsException(
-                    "More than 1 developer ID must be present in the request body to perform a merge.");
-        }
-        return developerManager.merge(mergeRequest.getDeveloperIds(), mergeRequest.getDeveloper());
+        return new ResponseEntity<>(HttpStatus.MOVED_PERMANENTLY);
```

Comment:
Just a clean up type thing that doesn't matter - I don't think this method still needs the `throws` clause.